lib/primitives/array-element.js

```diff
@@ -92,6 +92,15 @@ module.exports = function(BaseElement, registry) {
       return undefined;
     },
 
+    /*
+     * It reduces through a given array and calls get on each item.
+     */
+    getIn: function(paths) {
```

Comment:
And, any thoughts on better name? I can overload the `#get` method, but I'm not sure I like that.